vnext/JSI/Shared/JSI.Shared.vcxitems

```diff
@@ -17,6 +17,7 @@
     <ClCompile Include="$(MSBuildThisFileDirectory)ChakraRuntime.cpp" Condition="'$(OSS_RN)' != 'true'" />
   </ItemGroup>
   <ItemGroup>
+    <ClInclude Include="$(MSBuildThisFileDirectory)ByteArrayBuffer.h" />
```

Comment:
>"  [](start = 68, length = 2)

Should there be OSS_RN condition here too?

Reply:
No, all the OSS_RN condition in this vcxitems should go away eventually. Right now, we can't remove the rest yet due to dependencies on Microsoft/RN specific structures such as ScriptStore.

---
In reply to: [333117150](https://github.com/microsoft/react-native-windows/pull/3366#discussion_r333117150) [](ancestors = 333117150)